cmd/internal/login/login.go

```diff
@@ -24,28 +29,84 @@ API key information will be stored in ` + cfg.GetCredentialsConfigPath(),
 			APIKeySecret string
 		}{}
 
-		// Interactive prompt.
-		qs := []*survey.Question{
-			{
-				Name:   "APIKeyID",
-				Prompt: &survey.Input{Message: "API Key ID:"},
-			},
-			{
-				Name:   "APIKeySecret",
-				Prompt: &survey.Password{Message: "API Key Secret:"},
-			},
-		}
+		if !checkCreds {
```

Comment:
Nice to have: if credentials are being provided in the environment, warn the user that the credentials they are about to enter will have no effect.

Reply:
Added in 785f96c.